CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 0.13.4 (Unreleased)
 ===================
 
+associations
+---
+
+- MIRI MRS dedicated background exposures are now listed as science observations in
```

Comment:
Do the changes in #3542 really only affect MIRI MRS dedicated bkg exposures, or all dedicated bkg exposures? The particular example it was tested on is MIRI MRS, but hopefully it works for all.

Reply:
Oh, and you need a change log entry for this PR (#3611).

Reply:
Yes, this is only MIRI_MRS dedicated bkg exposures, this is a list so I can add any other exposure types needed.